EPRN.Portal/Controllers/PrnsController.cs

```diff
@@ -17,6 +26,8 @@ public PrnsController(IPRNService prnService)
         public async Task<IActionResult> Create(int? id)
         {
             var viewModel = await _prnService.CreatePrnViewModel();
+            viewModel.DateWithinDecemberRange = IsCurrentDateWithinDecOrJan();
```

Comment:
Can this also be set in the service please?

Reply:
Done.